sample-apps/delivery-customizations-sample-app/web/frontend/components/ErrorsBanner.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+import { Banner, List } from "@shopify/polaris";
+
+export function ErrorsBanner({ userErrors }) {
+  const titleContent = `To save this customization, ${
+    userErrors?.length > 1 ? `${userErrors.length} changes` : "1 change"
+  } needs to be made"`;
```

Comment:
I think the grammar here is incorrect in the non-one case. I believe it should be:

```suggestion
  const changesContent = userErrors.length === 1 ? "1 change needs" : `${userErrors.length} changes need`;
  const titleContent = `To save this customization, ${changesContent} to be made:`;
```